### ISSUE TITLE
Account 60 day ban

### ISSUE THREAD

**Issue by bgriffbass** [the-hideout/TarkovMonitor#23]
Got a 60 day ban for using the monitor, beware friends

**Comment by Razzmatazzz** [comment]
How do you know it was for using the monitor?

**Comment by GrantBirki** [comment]
@bgriffbass do you have any evidence to back this up? I'm not trying to call you out here but I am genuinely curious since I can not really imagine that BSG is issuing bans for something that isn't a cheat and was released yesterday.

Things that make me doubt this:

1. IIRC Battlestate Games doesn't issue "60 day bans". Its either all or nothing for someone suspected of "cheating" or "hacks"
2. This tool does not `write` or make **any** modifications to game files. It is simply monitoring the logs which the game creates. You could view the logs with a tool like **notepad** that is installed on every Windows machine. By this logic BSG should also ban people for having Windows installed
3. This tool was released _yesterday_. For BSG to ban players using this tool, they would need to already have something baked into their anti-cheat (like process / application signatures or heuristics) for this app. If BSG is doing this, instead of the grocery list of other items they have to do on launch day, I would be amazed

**TL;DR**: Without more evidence or information, I do not have any confidence that your presumed ban came from using this tool.
